src/pages/Categories/CategoriesBeta.vue

```diff
@@ -78,5 +73,48 @@ export default {
 		KvGrid,
 		KvPageContainer
 	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			categories: [],
+		};
+	},
+	methods: {
+		gatherCategoryData() {
+			this.apollo.query({
+				query: gql`query categoryRows {
```

Comment:
Do we want this query in a separate `.graphql` file?

Reply:
Inline queries are ok to use, especially when then are for a specialized case.